input/drivers/pygame_input.py

```diff
@@ -1,4 +1,4 @@
-
+import logging
```

Comment:
Dammit, I somehow missed that while refactoring - didn't test the emulator. I'm sorry, will now go through the code and see if there's anything else I might've missed =) Next time, please, don't bundle "this broke and needs to be fixed ASAP" fixes with unrelated pull requests, just send them as a separate PR - you can be sure I'll merge those as soon as I'll see them =)